dev_guide/builds.adoc

```diff
@@ -856,28 +856,39 @@ $ oc secrets new mysecret ca.crt=path/to/certificate
 $ oc secrets new mysecret ca.crt=path/to/certificate .gitconfig=path/to/.gitconfig
 ----
 ====
-
++
 [NOTE]
 ====
-Please note that SSL verification can be turned off, if `sslVerify=false` is set
-for the `http` section in your `.gitconfig` file:
+SSL verification can be turned off, if `sslVerify=false` is set for the `http`
+section in your `.gitconfig` file:
 ----
 [http]
         sslVerify=false
 ----
 ====
 
-. Add the `*secret*` to the builder service account. Each build is run with
+.  Add the `*secret*` to the builder service account. Each build is run with the
 `serviceaccount/builder` role, so you need to give it access your secret with
 following command:
-
 +
 ====
 ----
 $ oc secrets add serviceaccount/builder secrets/mysecret
 ----
 ====
 
+.  Add the `*secret*` to the `*BuildConfig*`:
+
+====
+source:
+  git:
+    uri: https://github.com/openshift/nodejs-ex.git
+  secrets:
+    - secret:
+        name: "mysecret"
+  type: Git
+----
+====
```

Comment:
@nak3 Thanks! Nice catch on the missing ----. For the sake of usability and saving the user an extra click, as well as remaining consistent with the rest of that section, I decided to maintain the code block. However, I provided a link to the Defining Secrets in the BuildConfig section for further reference.